src/style.css

```
@@ -1,3 +1,64 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  width: 100%;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  margin-left: 50px;
+  margin-right: 50px;
```

```suggestion
  padding: 0 50px;
```

add add `box-sizing: border-box` rule to * selector